lib/address.py

```diff
@@ -342,15 +371,9 @@ def to_scripthash_hex(self):
         '''Like other bitcoin hashes this is reversed when written in hex.'''
         return hash_to_hex_str(self.to_scripthash())
 
-    def to_P2PKH_script(self):
-        '''Return a P2PKH script.'''
-        return self.address.to_script()
-
-    def __str__(self):
-        return self.to_ui_string()
-
+    # no __str__ method
```

Comment:
Here is where I'm nervous.. hehe.  I'll try and see if anything expects the PublicKey output to have a str(e) -> hex bytes.  Hopefully not.